doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -27,4 +27,4 @@ Elasticsearch is a highly scalable open source full-text search and analytics en
 
 The JMS broker resource adapter enables your applications to communicate with any messaging provider. It specifies how components such as message-driven beans, Enterprise JavaBeans, and servlets can send or receive messages.
 
-
+High availability relies on a shared file system, a shared Elasticseach instance, and a shared JMS broker.
```

Comment:
I don't exactly agree with this statement.

ElasticSearch is not shared
JMS is also not exactly shared

The only shared is the file system, but can be also distributed in different ways:'
 - using git hooks to automatically backup content
 - using a SAN/NFS
 - using a distributed file system like Gluster